README.md

```diff
@@ -1,9 +1,29 @@
 # benchHackathonNov20
+This project is focused around developing both an iOS and Android app, using KMM for the shared logic between both platforms.
 
-## Mock-Server
+## The concept
+"CommunityKitchen" is a platform to donate and find excess food for those in need that would otherwise go to waste. 
+Currently, a user can: create an account and login and search for meals in their local area. 
 
-### Installing json-server
-We are currently using `json-server` as a mock server to run the project against.
+## Prerequisites
+
+### Android Studio
+Version 4.2 > is required to compile this project. From the latest changes, Arctic Fox (Canary 2, Beta) is required due to using Compose.
+
+### Xcode
+For running the iOS app via Xcode and for using the simulator, version 12.1 > is required.
+
+### Google Maps Api
+To use the location services and to view the available meals on the map, you will need to add the api key to your project.
```

Comment:
Where can I get the API key?

Reply:
Same as the Firebase project, someone can provide it :)

Reply:
So maybe it's worth mentioning where should I ask in order to get access to those files?

Reply:
@Gmiller290488 gave me access to Firebase. 
For Google Maps, I can give access to Google Cloud Console

Reply:
@fmontesino have added you to firebase.. Can you check if you have got access